package.json

```diff
@@ -7,31 +7,24 @@
     "url": "git@github.com:emberjs/ember-test-waiters.git"
   },
   "scripts": {
+    "build": "pnpm --filter '@ember/test-waiters' build",
     "lint": "pnpm --filter '*' lint",
     "lint:fix": "pnpm --filter '*' lint:fix",
-    "start": "concurrently 'npm:start:*' --restart-after 5000 --prefix-colors cyan,white,yellow",
+    "start": "concurrently \"pnpm:start:*\" --restart-after 5000 --prefix-colors cyan,white,yellow",
     "start:addon": "pnpm --filter @ember/test-waiters start -- --no-watch.clearScreen",
     "start:test-app": "pnpm --filter test-app start",
     "test": "pnpm --filter '*' test"
   },
   "devDependencies": {
+    "@glint/core": "^1.5.0",
```

Comment:
because of vscode reasons, some LSP/Tooling dependencies need to be added to the root package.json.

No CLI tool cares that these are here